Release 0.9.29

yenda opened the pull request:
- add validate method to StatusMessage protocol
- spec all message types for use in validate method
- use valid method after transit/decode step to reject
invalid messages

# steps to test
join #status and #test channel, client should display no error and no crash (there will be invalid command messages in the chat but they should trigger errors)

status: ready

Commit:
validate incoming message

- add validate method to StatusMessage protocol
- spec all message types for use in validate method
- use valid method after transit/decode step to reject
invalid messages

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

Commit:
[fix #6087] error on empty messages

add a guard so that `right-to-left-text?` function doesn't throw
an error on empty content

future empty messages aren't going to pass validation

mandrigin committed:
Fix version for 0.9.29
